### ISSUE TITLE
[Perception] What's the direction for this?

### ISSUE THREAD

**Issue by MicheleAWilliams-A11y** [w3c/wai-people-use-web-videos#97]
Admittedly, I found the written "Perception" resource a bit hard to follow, so that's making the video hard to process. 

I like the opening of the video where it explains "People perceive information through different senses. This includes perceiving information through hearing, seeing, and feeling." But overall I'm not sure how the audience is supposed to follow along or what they should try to take away. Even in these opening sentences it's not explained that people are using these senses in ways that compensate for an impairment in another sense. 

Overall I found myself wondering is this supposed to spur people into innovation, simply state what's currently available, get them to shift their mindsets about disability, or something else?

To me this is important to establish because it impacts the way you position the explanations. E.g., if this is just so people can answer a question such as "how do blind people read an online book" then you simply show a screen reader. But if it's to answer "how can I make this new experience I'm creating more inclusive to blind people" or "how do blind people experience such as visual world" then you delve deeper into translating visuals into text and haptics and use screen readers and Braille displays as just one example of tech's possibilities.

Overall the main (written) resource seems to go back-and-forth between specific examples and the broader concept. I think since we have resources like "Input" to go through the examples, we (in the videos and possibly in the text) should use this to talk about the broader concept of adaptation & adjustments; diversity, equity & inclusion; and generally not thinking something has to be one way.

**Comment by bakkenb** [comment]
Agree with these two points Michele is making...

> Even in these opening sentences it's not explained that people are using these senses in ways that compensate for an impairment in another sense.

> we (in the videos and possibly in the text) should use this to talk about the broader concept of adaptation & adjustments; diversity, equity & inclusion; and generally not thinking something has to be one way.

**Comment by nitedog** [comment]
This entire script has been heavily edited; please re-review.
